### PR TITLE
Add notice to join/leave message

### PR DESCRIPTION
**Pull request by RealYusufIsmail**
# PR Title
### Checklist

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines](/CONTRIBUTING.md).
- [x] I agree with the [license](/LICENSE).

### Changes
- [x] Refactoring
- [ ] Internal code
- [ ] External Code / Libraries
- [ ] Documentation
- [ ] Other: \_____

Closes Issue: -

## Description
Added notice to avoid any problems

### CODE REVIEW

File: src/main/java/com/github/black0nion/blackonionbot/systems/JoinLeaveSystem.java [hunk 1]
```diff
@@ -64,7 +65,13 @@ public void onGuildMemberJoin(final @NotNull GuildMemberJoinEvent event) {
 			final TextChannel channel = guild.getTextChannelById(id);
 			if (channel == null) return;
 			final File file = generateImage(Color.BLACK, author, guild, DrawType.JOIN);
-			channel.sendMessage(guild.getJoinMessage().replace("%userid%", author.getAsMention()).replace("%guildid%", guild.getEscapedName())).addFiles(FileUpload.fromData(file, "welcome.png")).queue();
```
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/systems/JoinLeaveSystem.java:67]: Please re-add this line, it's supposed to be a welcome message _but on the server_

File: src/main/java/com/github/black0nion/blackonionbot/systems/JoinLeaveSystem.java [hunk 2]
```diff
@@ -80,8 +87,12 @@ public void onGuildMemberRemove(final @NotNull GuildMemberRemoveEvent event) {
 			final TextChannel channel = guild.getTextChannelById(id);
 			if (channel == null) return;
 			final File file = generateImage(Color.BLACK, author, guild, DrawType.LEAVE);
-			channel.sendMessage(guild.getLeaveMessage().replace("%userid%", author.getAsMention()).replace("%guildid%", guild.getEscapedName()))
```
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/systems/JoinLeaveSystem.java:83]: same as above

**Review by SIMULATAN — CHANGES_REQUESTED** [review]
Please move the embed sending in front of the `if (id == -1) return;` as that'll disable the join message as well even though its unrelated

File: src/main/java/com/github/black0nion/blackonionbot/systems/JoinLeaveSystem.java (hunk 1 above)
Comment (RealYusufIsmail) [reply src/main/java/com/github/black0nion/blackonionbot/systems/JoinLeaveSystem.java:67]: what do you mean

Comment (SIMULATAN) [reply src/main/java/com/github/black0nion/blackonionbot/systems/JoinLeaveSystem.java:67]: the deleted line sends a message in a channel in the guild
the code you added is to send a DM to the user to make them aware of rules, links and that stuff
my plan was for both to co-exist

**Review by SIMULATAN — APPROVED** [review]
After further discussion, I figured out that I'm an idiot and the change is actually correct.
I'll test the change soon™ after which this PR will be ready to merge.

**Comment by RealYusufIsmail** [comment]
> After further discussion, I figured out that I'm an idiot and the change is actually correct. I'll test the change soon™ after which this PR will be ready to merge.

ok

**Pull request merged**